Dockerfile

```diff
@@ -17,7 +17,26 @@ WORKDIR /todo-app/todo_app
 
 ENTRYPOINT poetry run gunicorn wsgi:start -b 0.0.0.0:80
 
-FROM base as development
+FROM base as dev_base
 RUN poetry install
 
-ENTRYPOINT poetry run flask run --host=0.0.0.0
\ No newline at end of file
+FROM dev_base as development
+ENTRYPOINT poetry run flask run --host=0.0.0.0
+
+FROM dev_base as test
+ 
+# Install the long-term support version of Firefox (and curl if you don't have it already)
+RUN apt-get install -y firefox-esr curl
+  
+# Download geckodriver and put it in the usr/bin folder
+ENV GECKODRIVER_VER v0.30.0
+RUN curl -sSLO https://github.com/mozilla/geckodriver/releases/download/${GECKODRIVER_VER}/geckodriver-${GECKODRIVER_VER}-linux64.tar.gz \
+   && tar zxf geckodriver-*.tar.gz \
+   && mv geckodriver /usr/bin/ \
+   && rm geckodriver-*.tar.gz
```

Comment:
This is fine (and pretty good to know you can install things like this), but I  _think_ you may be able to use apt to install geckodriver